molecule/default/prepare.yml

```diff
@@ -28,3 +34,4 @@
     easyrsa_download: []
     easyrsa_download_pki: true
     easyrsa_download_dir: '~/'
+    ansible_python_interpreter: /usr/bin/python3.7
```

Comment:
The python interpreter must be specified to make the gather_facts pass. The problem is that in the file _/nkakouros.easyrsa/tasks/install.yml for molecule-test_ , which is executed after this, the Install dependencies task gives the future feature error. I don't know why this is the case since it was solved by using 

>     ansible_python_interpreter: /usr/bin/python3.7
Also, if I add the var to the task, I still get the same error: 
`- name: Install dependencies (Debian)
  apt:
    name: "{{ packages }}"
    update_cache: true
  when: ansible_os_family == 'Debian'
  vars:
    packages:
      - openssl
      - git

    interpreter_python: /usr/bin/python3.7`

![Screenshot 2024-06-19 231614](https://github.com/nkakouros-original/ansible-role-beats/assets/99359469/f9164a36-3338-4d5d-92c0-4991cdbe87ce)

But I tested with logging all the ansible facts, and the gather_facts works. 

![Screenshot 2024-06-19 231822](https://github.com/nkakouros-original/ansible-role-beats/assets/99359469/600e1d9c-48f2-4a9e-a65e-64dda11a3b8a)

I use the molecule.yml file that you provided last year,  https://github.com/nkakouros-original/linter-configs/blob/main/configs/molecule.yml.  When I use Ubuntu 20.04, or even 22.04, then I get the error  _Failed to create temporary directory.In some cases, you may have
been able to authenticate and did not have permissions on the
target directory. Consider changing the remote tmp path in
ansible.cfg to a path rooted in \"/tmp\", for more error
information use -vvv. Failed command was: ( umask 77 && mk_  , when running gather_facts. I have done a lot of debugging and research, but have not been able to solve it. So, I am wondering, is there any way we can skip over the ansible variables, like "ansible_os_family", so that we don't have to gather facts from the target system?

Reply:
I can not even run the roles I worked with last year, so it has to be version mismatches. I think I am using too new versions. Can you @nkakouros please send a screenshot of your configuration of the versions of the tools we are using, like the attached one above?